src/main/java/dk/aau/cs/model/tapn/TimedArcPetriNetNetwork.java

```diff
@@ -684,6 +684,16 @@ public Integer getVariableIndex(String name) {
         return null;
     }
 
+	public boolean doesSimilarRangeExist(ColorType newColorType) {
+		for (ColorType ct : colorTypes) {
+			if (ct.getColors().toString().equals(newColorType.getColors().toString()) && !ct.equals(newColorType)) {
```

Comment:
Is not there a better way to check that the integer ranges are the same, without converting to string comparison?

Reply:
If there is not a way to get the semantic value we should add it.